resources/js/src/app/components/common/TabList.js

```diff
@@ -0,0 +1,157 @@
+import { isDefined } from "../../helper/utils";
+
+const TabNavItem = {
+    name: "tab-nav-item",
+
+    render(createElement)
+    {
+        const anchorAttrs = {
+            role: "tab",
+            href: ""
+        };
+
+        if (this.tab.dataBuilderClickable)
+        {
+            anchorAttrs["data-builder-clickable"] = "";
+        }
+
+        const anchor = createElement(
+            "a",
+            {
+                staticClass: "nav-link",
```

Comment:
Gibt es eine schöne Möglichkeit, die CSS-Klassen von außen zu übergeben, sodass Theme-Entwickler die Möglichkeit hätten, das Styling anzupassen ohne das JavaScript verändern zu müssen?

Reply:
Also auf in die Componente verschachtelte elemente nur über eine prop von außen. Finde ich ok. Könnte ich gerne noch einbauen.